api/server/server.go

```diff
@@ -224,7 +230,12 @@ func NewServer(conf Config) (*Server, error) {
 	if err != nil {
 		return nil, fmt.Errorf("creating coreipfs: %s", err)
 	}
-	sched, err := scheduler.New(txndstr.Wrap(ds, "ffs/scheduler"), l, hs, cs)
+
+	var sr2rf func() (int, error)
+	if ms, ok := ms.(*sr2.MinerSelector); ok {
+		sr2rf = ms.GetReplicationFactor
+	}
```

Comment:
This should be temporary... but is for UX reasons, will explain later in review.